.circleci/config.yml

```diff
@@ -0,0 +1,38 @@
+version: 2.1
+
+executors:
+  release:
+    working_directory: ~/microservices-demo
+    docker:
+      - image: node:10
+
+jobs:
+  test-microservices-demo:
+    docker:
+      - image: circleci/node:9
+        user: node
+    working_directory: ~/microservices-demo
+    steps:
+      - checkout
+      - setup_remote_docker:
+          docker_layer_caching: false
+      - run:
+          name: Install Architect CLI
+          command: mkdir ~/.npm-global && npm config set prefix '~/.npm-global' && export PATH=~/.npm-global/bin:$PATH && source ~/.profile && npm install -g @architect-io/cli && architect --version
+      - run:
+          name: Architect Login
+          command: export PATH=~/.npm-global/bin:$PATH && source ~/.profile && architect --version && architect login -u $ARCHITECT_USERNAME -p $ARCHITECT_PASSWORD
+      - run:
```

Comment:
Make sure the push is only run on merges to master, and set the tag to `latest`

Reply:
This is currently valid for the published version of the CLI, and the reason the PR is a WIP is because those changes aren't going to work until we publish a new version